client/src/templates/Challenges/components/challenge-description.css

```
@@ -7,3 +7,9 @@
   margin-bottom: 1.45rem;
   font-size: 0.9rem;
 }
+
+#description p,
+#instructions p,
+#description ul {
+  font-family: 'Lato', sans-serif;
+}
```

```suggestion
#description p,
#instructions p,
#description ul,
#instructions ul {
  font-family: 'Lato', sans-serif;
}
```
If we are going this route, we need to also set this for the instructions section as well.